app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -126,6 +146,16 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         });
       }
 
+      function preventLinkFollowingForCurrentTab(event) {
```

Comment:
An example on GOV.UK where cmd+clicking acts as a toggle: https://www.gov.uk/guidance/the-highway-code/rules-for-pedestrians-1-to-35